src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -169,43 +174,47 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
+    const isCurrentPermTypeDelegated = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
+
     if (!panel) {
       columns.push(
         {
           key: 'consentDescription',
           name: messages.Description,
           fieldName: 'consentDescription',
           isResizable: true,
-          minWidth: (tokenPresent) ? 400 : 600,
-          maxWidth: (tokenPresent) ? 600 : 1000,
+          minWidth: (tokenPresent) ? ((isCurrentPermTypeDelegated) ? 400 : 800) : 600,
+          maxWidth: (tokenPresent) ? ((isCurrentPermTypeDelegated) ? 600 : 1300) : 1000,
           isMultiline: true
         }
       );
     }
 
-    columns.push(
-      {
-        key: 'isAdmin',
-        isResizable: true,
-        name: messages['Admin consent required'],
-        fieldName: 'isAdmin',
-        minWidth: (tokenPresent) ? 150 : 200,
-        maxWidth: (tokenPresent) ? 200 : 300,
-        ariaLabel: translateMessage('Administrator permission')
-      }
-    );
-
-    if (tokenPresent) {
+    if (isCurrentPermTypeDelegated) {
       columns.push(
         {
-          key: 'consented',
-          name: messages.Status,
-          isResizable: false,
-          fieldName: 'consented',
-          minWidth: 100,
-          maxWidth: 100
+          key: 'isAdmin',
+          isResizable: true,
+          name: messages['Admin consent required'],
```

Comment:
```suggestion
          name: translateMessage('Admin consent required'),
```
See https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/17#pullrequestreview-699168629 for a more detailed explanation of this